src/polycubed/src/config.cpp

```diff
@@ -64,6 +66,8 @@ static void show_usage(const std::string &name) {
             << ")" << std::endl;
   std::cout << "--configfile: configuration file (default: " << CONFIGFILE
             << ")" << std::endl;
+  std::cout << "--cubes-file: file to save last topology (default: " << LASTTOPOLOGYPATH
```

Comment:
What should so `--cubes-init`, create a new and empty cubes file?
Is this option really needed?, can't the user just remove the cubes file before running polyucubed?

Reply:
Clearing up the file does not look intuitive to me. I expect something like "start with a void config", which is exactly what I was suggesting. Open to suggestion for this new command line option.

Reply:
I don't know if that feature is really useful, anyway, I am not against it.

I have however a doubt about its behavior, What should happen if there is an already existing cubes file? Should it be cleaned up and then used to save the new topology?

Should we also provide an option to disable the saving of the cubes file? Just in case somebody doesn't need it and wants to avoid it's penalty.

Reply:
At the moment I implemented it (not pushed yet) without cleaning the file up. At the first action (for instance a creation of a cube) the file is overwritten, otherwise it remains with the previous configuration.

Reply:
> What should happen if there is an already existing cubes file? Should it be cleaned up and then used to save the new topology?

==> If you keep the state, the cubes.yaml file will be reinitialized and used to store the new topology (and config), i.e., the current state.


> Should we also provide an option to disable the saving of the cubes file? Just in case somebody doesn't need it and wants to avoid its penalty.

==> Very good point. I think "--cubes-nodump" can be a very good option then.
Thanks!